lib/FusionInventory/Agent/Task/Inventory/MacOS/CPU.pm

```diff
@@ -53,7 +53,8 @@ sub _getCpus {
     }
     close $handle;
 
-    my $type  = $sysprofile_info->{'Processor Name'} ||
+    my $type  = $sysctl_info->{'machdep.cpu.brand_string'} || 
```

Comment:
Sorry, I did not understand your at first about trim().
I can add $type =~ s/^\s+|\s+$//g line 79, but in glpi data without spaces. 
I could be wrong! 
Thank a lot, for your patience and good software!:)